src/PIL/ImageColor.py

```diff
@@ -101,6 +101,20 @@ def getrgb(color):
             int(rgb[2] * 255 + 0.5)
             )
 
+    m = re.match(r"hs[bv]\(\s*(\d+\.?\d*)\s*,\s*(\d+\.?\d*)%\s*,\s*(\d+\.?\d*)%\s*\)$", color)
```

Comment:
Here, and in others, looks like the initial opening, and final closing brackets are also redundant.

Shall we remove these to make the regex a bit shorter? The `m.group` calls can then begin from 0.

Reply:
The initial and final closing brackets are escaped - they are part of the string, e.g. `hsb(0,100%,50%)`

Reply:
Haha, we'd better keep them then!